app/src/main/jni/build_ffmpeg.sh

```diff
@@ -67,8 +75,6 @@ git checkout release/4.2
     --arch=arm \
     --cpu=armv7-a \
     --cross-prefix="${TOOLCHAIN_PREFIX}/armv7a-linux-androideabi16-" \
-    --nm="${TOOLCHAIN_PREFIX}/arm-linux-androideabi-nm" \
-    --strip="${TOOLCHAIN_PREFIX}/arm-linux-androideabi-strip" \
```

Comment:
What is the difference between llvm-strip and arm-linux-androideabi-strip ?
I have both in my system, the options seem somewhat different.
The build file I got from exoplayer, it is part of exoplayer I have modified slightly.
The next version of exoplayer changes this completely, ffmpeg is compiled in, instead of being linked.

Reply:
My android installation is brand new. I imported the project in android studio and installed all the required dependencies. `arm-linux-androideabi-strip` was not installed. 
The doc https://developer.android.com/ndk/guides/other_build_systems#autoconf shows that llvm-strip should be used.